packages/components/src/pages/SerializedSequence.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <VDiagramSequence :serialized-diagram="serializedDiagram" :interactive="interactive" />
+</template>
+
+<script>
+import VDiagramSequence from '@/components/DiagramSequence.vue';
+import { store } from '@/store/vsCode';
+
+export default {
+  name: 'v-serialized-sequence',
```

Comment:
I'm not sure about the name, `v-sequence-diagram` seems fine even if it's similar to `v-diagram-sequence`, at least it's a page vs. component.

The fact that the data (?) is serialized seems irrelevant to me.

Reply:
Yeah, I had `v-sequence-diagram` originally, but it seemed confusing to have that and `v-diagram-sequence`, but it is a better name so I'll change it back.

Reply:
Please change the file name too if you do that.